### PR TITLE
Update for Tutorial 5

### LINKED ISSUE #168: Update Tutorial 5 to Include Text Drawing
Upon completion of #161 , tutorial 5 needs to be updated to include appearing and disappearing of text.

### PR DESCRIPTION
**Pull request by TheCedarPrince**
**Link to relevant issue(s)**
Closes #168 

**How did you address these issues with this PR? What methods did you use?**
Added the ability to draw text to this animation.

### CODE REVIEW

**Comment by Wikunia** [comment]
This should normally go to v0.2. did you change it in master?

**Comment by TheCedarPrince** [comment]
Also, @Wikunia , I found a strange error/bug that is happening with my code. The gif draws the text the first time, but after the first drawing, the text is not redrawn. Here is the gif:

atomic

Here is the code:

```julia
using Animations
using Javis
using PeriodicTable
using Unitful

function ground(video, action, frame)
    background("white")
    sethue("black")
    text("$frame / 550", -240, -230)
end

function element(; color = "black")
    sethue(color)
    circle(O, 4, :fill)
    return val(:_current_scale)[1]
end

function info_box(args...; value = 0)
    fontsize(12)
    box(140, -210, 170, 40, :stroke)
    box(0, 175, 450, 100, :stroke)
    for element in elements
        if value == round(ustrip(element.atomic_mass))
            text(
                "Element name: $(element.name)",
                140,
                -220,
                valign = :middle,
                halign = :center,
            )
            text(
                "Atomic Radius: $(round(ustrip(element.atomic_mass)))",
                140,
                -200,
                valign = :middle,
                halign = :center,
            )
            textwrap("Description: $(element.summary)", 400, Point(-200, 125))
        end
    end
end

demo = Video(500, 500)
javis(
    demo,
    [
        BackgroundAction(1:550, ground),
        Action(
            1:550,
            :atom,
            (args...) -> element(),
            subactions = [
                SubAction(101:140, Scaling(1, 12)),
                SubAction(241:280, Scaling(12, 20)),
                SubAction(381:420, Scaling(20, 7)),
                SubAction(521:550, Scaling(7, 1)),
            ],
        ),
        Action(
            1:100,
            (args...) -> info_box(value = val(:atom)),
            subactions = [SubAction(1:30, sineio(), appear(:draw_text))],
        ),
        Action(
            141:240,
            (args...) -> info_box(value = val(:atom)),
            subactions = [SubAction(141:170, sineio(), appear(:draw_text))],
        ),
        Action(
            281:380,
            (args...) -> info_box(value = val(:atom)),
            subactions = [SubAction(281:310, sineio(), appear(:draw_text))],
        ),
        Action(
            421:520,
            (args...) -> info_box(value = val(:atom)),
            subactions = [SubAction(421:450, sineio(), appear(:draw_text))],
        ),
    ],
    pathname = "atomic.gif",
    framerate = 10,
)
```

Can you think of why the text in the top right box is not being displayed as a drawing each time I call the `info_box` action like I expect?

**Comment by Wikunia** [comment]
SubActions use relative frames so yours are never called.

File: test/unit.jl [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+        Leftover files from failed tests or errors can cause testing errors.
+        Therefore, we remove any files in the `images` directory not pertinent to testing before executing a test.
+        =#
+
+        EXCLUDE_FILES = [".keep"]
+        for f in readdir("images")
+            !(f in EXCLUDE_FILES) && rm("images/$f")
+        end
+        @test length(readdir("images")) == length(EXCLUDE_FILES)
+    end
```
Comment (Wikunia) [inline test/unit.jl:15]: This shouldn't appear here. I think you might have messed something up with your base branch. This should go into v0.2

File: docs/src/tutorials/tutorial_5.md [hunk 2]
```diff
@@ -372,20 +390,37 @@ javis(
...
                 SubAction(381:420, Scaling(20, 7)),
-                SubAction(521:550, Scaling(7, 1))
-            ]
+                SubAction(521:550, Scaling(7, 1)),
+            ],
+        ),
+        Action(
+            1:100,
+            (args...) -> info_box(value = val(:atom)),
+            subactions = [SubAction(1:30, sineio(), appear(:draw_text))],
```
Comment (Wikunia) [inline docs/src/tutorials/tutorial_5.md:407]: As this is a tutorial you should explain what's going on 😛

**Review by Wikunia — CHANGES_REQUESTED** [review]
Needs some more explanations and the bugfix.

File: docs/src/tutorials/tutorial_5.md (hunk 2 above)
Comment (TheCedarPrince) [reply docs/src/tutorials/tutorial_5.md:416]: Done.

File: test/unit.jl (hunk 1 above)
Comment (TheCedarPrince) [reply test/unit.jl]: I reverted this to v0.2 - not sure what happened? Might've been when I accidentally started off of master branch.

**Review by Wikunia — APPROVED** [review]
Lgtm

**Pull request merged**